cmd/commands/destroy_cluster.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+	"github.com/rook/kubectl-rook-ceph/pkg/crds"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+	"os"
+	"strings"
+)
+
+const (
+	destroyClusterQuestion = "Are you sure you want to destroy the cluster in namespace \"%s\"? "
+	destroyClusterAnswer   = "yes-really-destroy-cluster"
+)
+
+// DestroyClusterCmd represents the command for destroy cluster
+var DestroyClusterCmd = &cobra.Command{
+	Use:   "destroy-cluster",
+	Short: "delete all Rook CRDs",
+	Run: func(cmd *cobra.Command, args []string) {
+		question := fmt.Sprintf(destroyClusterQuestion, CephClusterNamespace)
+		confirmation, err := confirmPrompt(question, destroyClusterAnswer)
+		if err != nil {
+			logging.Fatal(err)
+		}
+
+		logging.Info(confirmation)
+		clientsets := GetClientsets(cmd.Context())
+		err = crds.DeleteCustomResources(cmd.Context(), clientsets, CephClusterNamespace)
```

Comment:
> Hey @subhamkrai I have a question here usually is good to have one single place to treat the errors that occurs inside the package I can change it of course but what do you think about that?.
> 
> By other side if I remove the return error, all my unittest will be useless since they cannot to use the return of the function to validate if works properly or not.
> 
> Let me know your thoughts

See the example for [RestoreQuorum()](https://github.com/rook/kubectl-rook-ceph/blob/master/pkg/mons/restore_quorum.go#L36-L43). The pattern this repo has is to have a method that is called that doesn't return an error, but there is a private method (lower case) that does return an error, which can be called by the unit tests. Does that work for you? It just keeps the cmd package simpler.

Reply:
thanks that works pretty good